Description: Disable the intersphinx plugin
 This is to avoid network access durring build.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-06-16

--- python-dogpile.cache-0.5.4.orig/docs/build/conf.py
+++ python-dogpile.cache-0.5.4/docs/build/conf.py
@@ -29,7 +29,7 @@ import dogpile.cache
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx',
+extensions = ['sphinx.ext.autodoc',
         'changelog', 'sphinx_paramlinks']
 
 changelog_sections = ["feature", "bug"]
